### PR TITLE
add demo of rgb-node:0.1

### PR DESCRIPTION
**Pull request by St333p**
Here is my proposal for a written guide that aims at reproducing the video demo on youtube. The structure is similar to the one I did for lnp-node.

### CODE REVIEW

**Review by dr-orlovsky — APPROVED** [review]

**Comment by dr-orlovsky** [comment]
@UkolovaOlga can you also pls review?

**Comment by dr-orlovsky** [comment]
And does anybody has the idea why the CI fails?

**Comment by zoedberg** [comment]
> And does anybody has the idea why the CI fails?

it's missing `apt-get update`, #105 fixes that

File: doc/demo-0.1/README.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+*Note:* to stop/start/restart containers you can use `docker-compose stop/start/restart`
+
+Finally we can setup aliases to ease calls to command-line interfaces:
+```bash=
+alias rgb0-cli="docker exec -it rgb-node-0 rgb-cli"
+alias rgb1-cli="docker exec -it rgb-node-1 rgb-cli"
+```
+
+## Demo
+In this demo, `rgb-node-0` takes the part of the issuer and transfers some of the newly minted asset to a user, `rgb-node-1`.
```
Comment (UkolovaOlga) [inline doc/demo-0.1/README.md:71]: >`rgb-node-0` takes the part of the issuer 

better to say 'acts as an issuer'

>minted asset to a user, `rgb-node-1`.

'the' user, not 'a'

File: doc/demo-0.1/README.md [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+
+In order to get an idea of the functionality exposed by `rgb-cli`, you can run for instance:
+```bash=
+rgb0-cli help
+rgb0-cli fungible help
+rgb0-cli fungible help list
+rgb0-cli genesis help
+```
+### premise
```
Comment (UkolovaOlga) [inline doc/demo-0.1/README.md:80]: Premise with capital 'P'

File: doc/demo-0.1/README.md [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+
+RGB-node does not handle wallet-related functionalities, it just performs RGB-specific tasks over data that will be provided by an external wallet such as [bitcoind](https://github.com/bitcoin/bitcoin). In particular, in order to demonstrate a basic workflow with issuance and a transfer, we will need:
```
Comment (UkolovaOlga) [inline doc/demo-0.1/README.md:82]: >RGB-node does not handle wallet-related functionalities

'functionality' - singular form

>in order to demonstrate a basic workflow with issuance and a transfer,

...and transfer, no 'a'

File: doc/demo-0.1/README.md [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+- an `issuance_utxo` to which `rgb-node-0` will bind newly issued asset
```
Comment (UkolovaOlga) [inline doc/demo-0.1/README.md:83]: the newly issued asset

File: doc/demo-0.1/README.md [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+- a `change_utxo` on which `rgb-node-0` receives asset change
```
Comment (UkolovaOlga) [inline doc/demo-0.1/README.md:84]: the asset

File: doc/demo-0.1/README.md [hunk 6]
```diff
@@ -0,0 +1,161 @@
...
+- a `receive_utxo` on which `rgb-node-1` receives asset
```
Comment (UkolovaOlga) [inline doc/demo-0.1/README.md:85]: the asset

File: doc/demo-0.1/README.md [hunk 7]
```diff
@@ -0,0 +1,161 @@
...
+rgb1-cli fungible invoice <asset-id> 100 79d0191dab03ffbccc27500a740f20a75cb175e77346244a567011d3c86d2b0b:0
+# example output:
+# [...]
+# Invoice: rgb20:utxob10gz0mmkpn2jykqkdwpjeltfwptkfke00zsgmz8lvu7ffsschq0sqxwjnq4?asset=rgb1yprwyam35r0varfhtswcawkhwdhm025mzd5qcejeun0kqj0dtqyqpjn5w0&amount=100
+# Outpoint blinding factor: 6754896042118498142
+```
+To be able to accept transfers related to this invoice, we will need the original `receive_utxo` and the `blinding_factor` that was used to include it in the invoice.
+
+### Transfer
+To transfer some amounts of asset to `rgb-node-1` to pay the new invoice, `rgb-node-0` needs to create a consignment and commit to it into a bitcoin transaction. So we will need the invoice and a partially signed bitcoin transaction that we will modify to include the commitment. Furthermore, `-i` and `-a` options allow to provide an input utxo from which to take asset and an allocation for the change in the form `<amount>@<utxo>`.
```
Comment (UkolovaOlga) [inline doc/demo-0.1/README.md:133]: >some amounts of **the** asset to

>utxo from which to take **the** asset

**Review by UkolovaOlga — CHANGES_REQUESTED** [review]
Minor linguistic changes required

**Review by UkolovaOlga — APPROVED** [review]
Ready for merging

**Pull request merged**